mono/metadata/opcodes.h

```diff
@@ -23,10 +23,15 @@ MONO_BEGIN_DECLS
 #define OPDEF(a,b,c,d,e,f,g,h,i,j) \
 	MONO_ ## a,
 
-typedef enum {
+#ifndef __MONO_METADATA_BASIC_BLOCK_H__
```

Comment:
What's so broken with mono that we need this?

Reply:
Some combination of things.
Public header to use internal enum I think.